pattern-1/bosh-release/README.md

```diff
@@ -1,242 +1,96 @@
 
-12. Download latest bosh-lite warden stemcell from bosh.io and upload it to BOSH Director.
-
-        if [ ! -f bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz ]; then
-            echo -e "\e[32m>> Stemcell does not exist! Downloading... \e[0m"
-            wget https://s3.amazonaws.com/bosh-core-stemcells/warden/bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-        fi
-
-        echo -e "\e[32m>> Uploading Stemcell... \e[0m"
-        bosh -e vbox upload-stemcell bosh-stemcell-3445.7-warden-boshlite-ubuntu-trusty-go_agent.tgz
-
-13. Deploy the WSO2 API Manager BOSH release manifest in BOSH Director.
-
-        echo -e "\e[32m>> Deploying bosh release... \e[0m"
-        yes | bosh -e vbox -d wso2apim deploy wso2apim-manifest.yml
-
-14. Add route to VirtualBox network.
-
-        echo -e "\e[32m>> Adding route... \e[0m"
-        sudo route add -net 10.244.0.0/16 gw 192.168.50.6
-
-15. List VMs.
-
-        echo -e "\e[32m>> Listing VMs... \e[0m"
-        bosh -e vbox vms
-
-
-[1]: http://bosh.io
-[2]: https://github.com/bhathiya/pivotal-cf-apim
-[3]: https://github.com/cloudfoundry/bosh-deployment 
+4. Execute the deploy.sh script.
+   ```
+   ./deploy.sh
+   ```
+   Executing this script will setup MySQL, BOSH environment and will deploy WSO2 API Manager 2.1.0 deployment pattern 1 on BOSH director.
 
+5. Find the IP addresses of created VMs via the BOSH CLI and access the WSO2 API Manager Publisher, Store and Management Console via a web browser.
+    ```
+    bosh -e vbox vms
+    ...
+    
+    Deployment 'wso2apim'
+    
+    Instance                                                 Process State  AZ  IPs          VM CID                                VM Type  
+    wso2apim_1/da718160-7588-4594-a6fb-71aa73845a1b          running        -   10.244.15.2  45149778-a699-4030-7374-dd9613f43901  wso2apim-resource-pool  
+    wso2apim_2/25e4cbc3-061d-48c9-ba33-5d71edb74f29          running        -   10.244.15.3  cb0408a3-50ed-4d7d-66ed-2ef50c1a2dd0  wso2apim-resource-pool  
+    wso2apim_analytics/04d6fb22-82bd-42d6-9e93-a42ad5f6ec8d  running        -   10.244.15.4  f8bf7609-eeab-4700-4c08-70bf0edfd18a  wso2apim_analytics-resource-pool  
+    
+    3 vms
+    
+    Succeeded
+    ...
+    ```
+    To ssh to the instance
+    ```
+    bosh -e vbox -d wso2apim ssh <instance_id>
+    e.g. bosh -e vbox -d wso2apim ssh wso2apim_1/da718160-7588-4594-a6fb-71aa73845a1b
+    ```
+    Access the Publisher, Store and Management Console
+    ```
+    WSO2 API Manager Publisher: https://10.244.15.2:9443/publisher
+    WSO2 API Manager Store: https://10.244.15.2:9443/store
+    WSO2 API Manager Management Console: https://10.244.15.2:9443/carbon/
+    ```
+
+# Additional Info
+
+Structure of the files of this repository will be as below :
+```
+└── bosh-release
+    ├── config
+    ├── dbscripts
+    ├── deployment
+    ├── dist
+    ├── jobs
+    ├── packages
+    ├── src
+    ├── create.sh
```

Comment:
I think it's better to remove the helper bash scripts and just include files required for building the BOSH release, CF Tile and the Service Broker in this repository.